src/game_main.cpp

```diff
@@ -217,6 +256,138 @@ std::optional<FpsLimiter> createLimiter(const data::GameOptions& options) {
   }
 }
 
+
+std::string effectiveGamePath(
```

Comment:
As it chooses between option provided gamepath and userprofile provided game path, a comment would be better in explaining that.

Reply:
Is https://github.com/lethal-guitar/RigelEngine/pull/495/commits/b89674100d0d7e9ad9ac5b82e7ebbef5f1173e76 roughly what you had in mind?

Reply:
indeed. thanks for adding it